packages/vike-react/renderer/getLang.ts

```diff
@@ -0,0 +1,42 @@
+export { getLang }
+
+import type { PageContext } from 'vike/types'
+import { isCallable } from './utils/isCallable.js'
+
+/**
+ * Get the page's lang if defined, either from the config, the additional data fetched by
+ * the page's data() and onBeforeRender() hooks or from other hooks.
+ */
+function getLang(pageContext: PageContext): null | string {
+  // from data() hook
+  if (pageContext.data?.lang !== undefined) {
+    return pageContext.data.lang
+  }
```

Comment:
Maybe we don't need to support this as `lang` isn't something a user would want to data-fetch?

Reply:
Yeah, I'm not sure too, removed for now.